templates/desktop/tenders/contact-us.html

```diff
@@ -1,146 +0,0 @@
-{% extends "desktop/base_desktop.html" %}
```

Comment:
Oh no sorry I see, it redirects to `/desktop/tenders-contact-us.html`. Fair enough.